client/styles.scss

```diff
@@ -1,4 +1,4 @@
-$path-assets: '../assets';
+$path-assets: './assets';
```

Comment:
haha. moving the assets folder to client has nothing to do with auth and the user model. (but okay haha)